File: 23_C#Basics/_05_LoopsPartTwo/_01_ReadText/_01_ReadText.cs
using System;

public class _01_ReadText
{
    public static void Main(string[] args)
    {
        // Use a while loop to continuously read input until "Stop" is entered
        while (true)
        {
            string input = Console.ReadLine();

            if (string.Equals(input, "Stop", StringComparison.OrdinalIgnoreCase))
            {
                break;
            }

            // Print the input to the console
            Console.WriteLine(input);
        }
    }
}

File: 01_ProgrammingBasics/_05_LoopsPartTwo/_01_ReadText.java
package _01_ProgrammingBasics._05_LoopsPartTwo;

import java.util.Scanner;

public class _01_ReadText {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Use a while loop to continuously read input until "Stop" is entered
        while (true) {
            String input = scan.nextLine();

            if (input.equalsIgnoreCase("Stop")) {
                break;
            }

            // Print the input to the console
            System.out.println(input);
        }

        scan.close();
    }
}
